From: Simon Chopin <simon.chopin@canonical.com>
Date: Fri, 10 Apr 2026 05:38:47 +0000
Subject: Extract the version from the Debian changelog

Forwarded: not-needed
Last-Update: 2022-03-01

Upstream uses the VCS to get the correct version in the build system, but we're using tarballs.
This patch uses a similar mechanism but extracts the version from the packaging instead.
Last-Update: 2022-03-01
---
 bootstrap | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bootstrap b/bootstrap
index f70ff03..3e39644 100755
--- a/bootstrap
+++ b/bootstrap
@@ -1,7 +1,7 @@
 #!/bin/sh
 set -e
 
-git describe --tags --always --dirty > VERSION
+dpkg-parsechangelog -S Version | sed 's/\(.*\)-[^-]*/\1/' > VERSION
 
 # generate list of source files for use in Makefile.am
 # if you add new source files, you must run ./bootstrap again
